packages/design-system/src/components/OcFilterChip/OcFilterChip.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { defaultPlugins, mount } from 'web-test-helpers'
+import OcFilterChip from './OcFilterChip.vue'
+
+const selectors = {
+  clearBtn: '.oc-filter-chip-clear'
+}
+
+describe('OcFilterChip', () => {
+  it('renders the filterName when no selected item has been given', () => {
+    const { wrapper } = getWrapper()
+    expect(wrapper.html()).toMatchSnapshot()
```

Comment:
I'd like to avoid a snapshot test for such a case, because it  fails/needs to be updated when something unrelated in the component has changed. Could you `find` the element on the wrapper and do a string match instead?